clients/unshaded/src/main/java/tachyon/client/file/options/InStreamOptions.java

```diff
@@ -50,6 +55,16 @@ public Builder(TachyonConf conf) {
       ReadType defaultReadType =
           conf.getEnum(Constants.USER_FILE_READ_TYPE_DEFAULT, ReadType.class);
       mTachyonStorageType = defaultReadType.getTachyonStorageType();
+      try {
+        mLocationPolicy =
+            CommonUtils
+                .createNewClassInstance(
+                    ClientContext.getConf().<FileWriteLocationPolicy>getClass(
+                        Constants.USER_FILE_WRITE_LOCATION_POLICY),
+                    new Class[] {}, new Object[] {});
+      } catch (Exception e) {
```

Comment:
hmm, I think my previous comment was lost due to some reason. I don't want the builder function to throw many exceptions, and therefore complicate the interface.
But I will add some message.